README.md

```diff
@@ -285,3 +285,21 @@ Debug Playwright tests with the GUI tool that can be opened with
 
 Update Playwright to the latest version with
 `npm install -D @playwright/test@latest`
+
+## Accessibility testing
+
+This project integrates @axe-core/playwright to automatically scan for accessibility violations and ensure our application meets WCAG standards.
+
+Installation
+
+If you are setting up the project for the first time, install the accessibility testing plugin:
+
+`npm install --save-dev @axe-core/playwright`
```

Comment:
Is this needed or does the general `npm install` handle dev deps for us too? cc @Toxiapo because Im not 100% sure